src/drive/web/modules/drive/SharingsContainer.jsx

```diff
@@ -25,13 +25,23 @@ class SharingFetcher extends React.Component {
       const resp = await client
         .collection('io.cozy.files')
         .all({ keys: sharedDocuments })
+
+      const parent_ids = resp.data.map(f => f.dir_id)
```

Comment:
Nitpick:

```suggestion
      const parentIds = resp.data.map(f => f.dir_id)
```

To stay consistent with the casing :)